scripts/google-alert.js

```diff
@@ -0,0 +1,9 @@
+const info = {
+  name: "Google Detector",
+  description: "Alerts the user when you are at google.ca",
+  match: "https://www.google.ca",
+};
+
+(function () {
```

Comment:
Here I'd tell you to use the Arrow Functions: https://www.freecodecamp.org/news/when-and-why-you-should-use-es6-arrow-functions-and-when-you-shouldnt-3d851d7f0b26/